#!/usr/bin/env python
PACKAGE = "lvt2calib"

from math import pi

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("minCircularity", double_t, 0, "min circularity of blob", 0.8, 0, 1)
gen.add("minInertiaRatio", double_t, 0, "min inertia ratio of blob", 0.1, 0, 1)
gen.add("minArea", double_t, 0, "min area of blob", 50, 0, 500)

exit(gen.generate(PACKAGE, "lvt2calib", "Camera"))
